.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Template for requesting new features
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+## Context
+
+What is the problem that you are trying to solve?
+Why is this problem relevant?
+
+## Solution
+
+(This section and the ones below are optional.)
+What are the possible solutions?
+If there are multiple alternatives, what are the benefits and drawbacks of each one?
+
+## Subtasks
+
+- [ ] If there is a solution, what are the subtasks for completing this issue?
```

Comment:
if a person creates a feature request does this necessarily mean that this person is a good one to decompose it? shouldn't the decomposition be done by the implementor of the feature?